StackoverflowChatbot/CommandProcessors/PriorityProcessor.cs

```diff
@@ -94,15 +94,14 @@ public bool ProcessCommand(EventData data, out IAction? action)
 				return true;
 			}	
 
-			// Why is action getting assigned but not unused?
+			// Why is action getting assigned but is unused?
 			action = null;
 			return false;
 		}
 
 		public bool TryGetNativeCommands(string key, out Type? value) => _nativeCommands.TryGetValue(key, out value);
 		public IEnumerable<string> NativeKeys => _nativeCommands.Keys;
 
-		//NativeKeys, 
```

Comment:
In the other pull request it's gone.

Reply:
Uhm, are you referring to the comments?

Reply:
yes. But I think I also merged the `TryGetNativeCommand` with `ProcessCommand`, because the `ProcessCommand` could be shortened to nearly nothing

Reply:
I'm not sure I follow. What should be done here?